@here/harp-mapview/package.json

```diff
@@ -47,17 +47,25 @@
     "devDependencies": {
         "@here/harp-test-utils": "^0.11.0",
         "@types/chai": "^4.1.2",
+        "@types/copy-webpack-plugin": "^5.0.0",
+        "@types/html-webpack-plugin": "^3.2.1",
         "@types/mocha": "^5.2.7",
         "@types/node": "^12.0.8",
         "@types/sinon": "^7.0.13",
+        "@types/webpack": "^4.41.0",
+        "@types/webpack-merge": "^4.1.5",
         "chai": "^4.0.2",
         "copyfiles": "^2.0.0",
+        "copy-webpack-plugin": "^5.0.5",
         "cross-env": "^6.0.3",
+        "html-webpack-plugin": "^3.2.0",
         "mocha": "^6.1.4",
         "sinon": "^7.1.1",
         "source-map-support": "^0.5.2",
         "three": "^0.110.0",
-        "typescript": "^3.7.2"
+        "typescript": "^3.7.2",
+        "webpack": "^4.41.2",
```

Comment:
Putting all the deps here, doesn't help user (one that npm-requires `@here/harp-mapview`) as it will not have `devDependencies` installed.

Adding this do `dependencies` is also no-go as we don't want user to be forced to use webpack.

I think, this script should either declare all these deps as `peerDependecies` or we should create separate package like mentioned somewhere `@here/harp-webpack-utils` or something.

Reply:
ok, actually I had it in a separate package at first. Was trying to simplify things but you are right I overlooked that dev dependencies are not installed transitively. Will move back to separate package.